google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -15,12 +15,15 @@
  */
 package com.google.cloud.bigtable.data.v2;
 
+import com.google.api.client.util.Preconditions;
```

Comment:
Please use `import com.google.common.base.Preconditions;` instead (that should fix the dependency build failure).